queries.yml

```
@@ -1,2 +1,23 @@
+  - 5585166
+  - 5589075
+  - 5589575
+  - 5594273
+  - 5595139
+  - 5600599
+  - 5600619
+  - 5600664
+  - 5605080
+  - 5605555
+  - 5605600
+  - 5605647
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)